src/services/admin-api.ts

```diff
@@ -153,7 +153,7 @@ export default class AdminApi {
     return balanceMiddleware.getStatus()
   }
 
-  private async postBalance (url: string, _data: object) {
+  private async postBalance (url: string | undefined, _data: object) {
```

Comment:
Why `| undefined`?

Reply:
`req.url` is `string | undefined` now, so the url params need to become `string | undefined`